py-polars/polars/utils/decorators.py

```diff
@@ -69,6 +69,41 @@ def _rename_kwargs(
             kwargs[new] = kwargs.pop(alias)
 
 
+def deprecated_default_value_kwargs(
```

Comment:
I realise this is still in 'draft' mode, but just wanted to check something - does using this decorator affect the function's docstring? What about it's signature, what would it show as the type of a deprecated argument?

Reply:
I have gone with just clarifying docstrings for now, so the decorator is gone.